### ISSUE TITLE
include personnel identifier 

### ISSUE THREAD

**Issue by ferrighi** [metno/mmd#260]
Decide how to store personal identifiers, e.g. OrcID in mmd. This is often supplied in the creator_url in netCDF files.

**Comment by steingod** [comment]
We need to update ACDD guidnace as well. Preferably with identifier in
paranthesis.

On Wed, 2024-09-11 at 03:03 -0700, Lara Ferrighi wrote:
> Decide how to store personal identifiers, e.g. OrcID in mmd. This is
> often supplied in the creator_url in netCDF files.
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this
> thread.Message ID: ***@***.***>

-- 
Dr. Øystein Godøy
Norwegian Meteorological Institute 
P.O.BOX 43, Blindern, N-0313 OSLO, Norway
Ph: (+47) 9802 4433

**Comment by mortenwh** [comment]
except - why "pid" at the end. I think other fields use just `<field>_id`, no? Does it matter?

**Comment by mortenwh** [comment]
No, you're right. I mixed up with ACDD... We have "metadata_identifier" and "alternate_identifier" but I guess we don't need to worry about this.
